iree/hal/local/inline_command_buffer.c

```diff
@@ -383,12 +380,16 @@ static iree_status_t iree_hal_inline_command_buffer_dispatch(
       iree_hal_local_executable_cast(executable);
   iree_hal_local_executable_layout_t* local_layout =
       local_executable->executable_layouts[entry_point];
+  iree_host_size_t local_memory_size =
+      local_executable->dispatch_attrs
+          ? local_executable->dispatch_attrs[entry_point].local_memory_size *
+                IREE_HAL_WORKGROUP_LOCAL_MEMORY_PAGE_SIZE
```

Comment:
wondering about units and naming here

would it be more accurate to use `local_memory_pages` instead of `local_memory_size`?

Reply:
good idea! that's much clearer